api/proto/info/info.proto

```diff
@@ -63,7 +62,7 @@ enum ZInfoTypes {
 }
 
 // XXX duplicate from devmodel.proto:
-// Needs to have a separate name
```

Comment:
That comment is still valid. So just add your new comment.

Reply:
Ah I see what u mean.. IPhyIoType Vs. PhyIoType. Changed it to:

// Needs to have a separate name than the one in devmodel.proto